src/core/server/saved_objects/service/saved_objects_client.ts

```diff
@@ -193,6 +195,13 @@ export interface SavedObjectsAddToNamespacesOptions extends SavedObjectsBaseOpti
   refresh?: MutatingOperationRefreshSetting;
 }
 
+export interface SavedObjectsAddToWorkspacesOptions extends SavedObjectsBaseOptions {
```

Comment:
Since there is already one workspaces argument in `addToWorkspace`, do we need to omit workspaces options here?

Reply:
I think they are different. workspaces within `addToWorkspace` method arguments, it's the target workspaces that saved objects want to added to.  `workspaces` in this interface `SavedObjectsBaseOptions` is more like used for validating source saved objects are exists in those workspaces.

Reply:
The `options.workspaces` was not been used in `src/core/server/workspaces/saved_objects/workspace_saved_objects_client_wrapper.ts` or `src/core/server/saved_objects/service/lib/repository.ts`. Is this by intention?

Reply:
moved the source workspace to the `options.workspaces` to align with other method in `src/core/server/saved_objects/service/lib/repository.ts` class